site/pages/docs/browser-support-en.hbs

```diff
@@ -0,0 +1,18 @@
+---
+{
+	"title": "Browser Support",
+	"language": "en",
+	"description": "Browser support for the Web Experience Toolkit.",
+	"altLangPrefix": "index",
```

Comment:
This is the reason why it's not failing. It's pointing to another page which might actually be worse. I would recommend copying the english page, wrapping in a `<div lang="en"></div>` in the meantime and change this value to `browser-support`